### PR TITLE
[MODULAR] Re-adds the fancy kimono (new sprite)

### PR DESCRIPTION
**Pull request by YakumoChen**
## About The Pull Request

I went to play my character best known for their fancy kimono and found out it was gone! What the hell? Put that shit back.
Partially resprited by @Ebin-Halcyon 

image
image
image
image

## How This Contributes To The Skyrat Roleplay Experience

It has a unique sprite that the recolourable yukata can't replicate. Also, yukatas aren't kimonos. 

## Changelog

:cl:
add: Readds the fancy-looking kimono that looked different from the others.
/:cl:

### CODE REVIEW

**Comment by glitchbow** [comment]
THANK YOU, I missed it too.

**Comment by Nerev4r** [comment]
For the sake of transparency, this item was intentionally removed in https://github.com/Skyrat-SS13/Skyrat-tg/pull/16072.

**Comment by YakumoChen** [comment]
The PR body already addresses that

File: modular_skyrat/master_files/code/modules/clothing/under/costume.dm [hunk 1]
```diff
@@ -99,6 +99,15 @@
 	desc = "A traditional ancient Earth Japanese Kamishimo."
 	icon_state = "kamishimo"
 
+/obj/item/clothing/under/costume/skyrat/kimono_fancy
+	icon = 'modular_skyrat/master_files/icons/obj/clothing/under/costume.dmi'
+	worn_icon = 'modular_skyrat/master_files/icons/mob/clothing/under/costume.dmi'
+	name = "fancy kimono"
+	desc = "A traditional ancient Earth Japanese Kimono. Longer and fancier than a yukata."
+	icon_state = "fancy_kimono"
```
Comment (tf-4) [inline modular_skyrat/master_files/code/modules/clothing/under/costume.dm]:
```suggestion
	name = "fancy kimono"
	desc = "A traditional ancient Earth Japanese Kimono. Longer and fancier than a yukata."
	icon_state = "fancy_kimono"
	icon = 'modular_skyrat/master_files/icons/obj/clothing/under/costume.dmi'
	worn_icon = 'modular_skyrat/master_files/icons/mob/clothing/under/costume.dmi'
```

**Review by tf-4 — CHANGES_REQUESTED** [review]
Just one very minor adjustment from me

**Review by tf-4 — APPROVED** [review]

**Review by Ebin-Halcyon — CHANGES_REQUESTED** [review]
Personally against this, for the same reason as Orion has put it in the discord, far too large, it's in the uniform slot which hands overlay over and the sprite quality is generally low quality, if you want to give it a shot and try to resprite it to be better, be my guest.

Another suggestion if you do up the quality on it is moving it over to the suit slot so the hands aren't overlayed on the uniform.

**Comment by YakumoChen** [comment]
@Ebin-Halcyon I'm not a spriter, I have no idea where to even start with the sprite because I don't see what's low quality about it
I can move it to the suit slot pretty easily but I really don't see where the 'low quality' claim comes from. Not really a fair ask.

**Comment by YakumoChen** [comment]
image
like, it's got some basic shading already, there's nothing out of place besides the fact it's a fairly big sprite but, that's not a huge deal?

**Comment by Ebin-Halcyon** [comment]
> @Ebin-Halcyon I'm not a spriter, I have no idea where to even start with the sprite because I don't see what's low quality about it I can move it to the suit slot pretty easily but I really don't see where the 'low quality' claim comes from. Not really a fair ask.

A quick list off of the low quality things of the original sprite
- Lack of outlines
- Mediocre colors
- strangely asymmetrical shading on the arms and other places

Sprites that go more than 3+ pixels off the sprite also look very garish and offputtingly large in game, ala the old victorian dresses.
In the end I can't close or merge the PR itself right now, so it's up to other maintainers. I'm just not a fan of the overall quality.

**Comment by glitchbow** [comment]
> Sprites that go more than 3+ pixels off the sprite also look very garish and offputtingly large in game, ala the old victorian dresses. In the end I can't close or merge the PR itself right now, so it's up to other maintainers. I'm just not a fan of the overall quality.

Is that why Victorian dresses were removed? I find that ridiculous. I liked both the fancy kimono and the Victorian dress, the latter of which I would also like put back in but I'm new to formatting sprites for BYOND. This is something I would be willing to learn to sprite for, honestly, especially considering how much I value the character I designed with that dress. To me, these sound a little less like "quality" issues and more of difference in personal tastes. I think similar comments were once made about Tallborgs being garish and yet they're the most commonly played cyborg type in the server now.

Ideally, a good fix for the fancy Kimono would to make it colorable, outlined a little better, and _maybe_ edit the sleeve asymmetry, although you can see from the front it is a patterned, asymmetrical style dress. I assume that's what you meant by "other places". ~~Might as well PR to remove Vertical Split on Hair Gradient and Heterochromatic as a trait if we want to complain about sprite symmetry, pff.~~ I really do dislike the removal of content over the adjustment of it.

If the suit slot is needed to justify the pixel size, I'm fine with that as well. It wouldn't be realistic to put a coat over an item like this anyhow.

**Comment by YakumoChen** [comment]
redoing it in #17196 because of merge conflicts

**Pull request closed without merging**